exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/TraceTranslatorTest.java

```diff
@@ -0,0 +1,205 @@
+        .putAttributeMap("/http/status_code", AttributeValue.newBuilder().setBoolValue
+            (true).build())
+        .putAttributeMap("anotherKey", AttributeValue.newBuilder().setIntValue
+            (100).build())
+        .putAttributeMap("g.co/agent", AttributeValue.newBuilder().setStringValue
+            (TruncatableString.newBuilder().setValue("opentelemetry-java [0.6.0]").setTruncatedByteCount(0)).build());
+    for (Map.Entry<String, AttributeValue> entry : fixedAttributes.entrySet()) {
+      attributesBuilder.putAttributeMap(entry.getKey(), entry.getValue());
+    }
+
+    Span.Attributes allAtributes = attributesBuilder.build();
+
+    assertEquals(allAtributes, TraceTranslator.toAttributesProto(attributes, fixedAttributes));
+
+  }
+
+  @Test
+  public void toTimeEventsProtoTest(){
+    List<SpanData.Event> events = new ArrayList<>();
+    SpanData.Event eventOne = new SpanData.Event() {
+      // The SpanData.Event interfaces requires us to override these four methods
+      @Override
+      public long getEpochNanos() {
+        return 0;
+      }
+
+      @Override
+      public int getTotalAttributeCount() {
+        return 0;
+      }
+
+      @Override
+      public String getName() {
+        return "eventOne";
+      }
+
+      @Override
+      public Attributes getAttributes() {
+        return Attributes.newBuilder().setAttribute("key", "value").build();
+      }
+    };
+    events.add(eventOne);
+
+    Span.TimeEvents timeEvents = TraceTranslator.toTimeEventsProto(events);
+    assertEquals(1, timeEvents.getTimeEventCount());
+    //TODO: Figure out how to use timeEvents to call SpanData.Event methods to ensure those are working properly
+  }
+
+  @Test
+  public void toStatusProtoTest(){
+    io.opentelemetry.trace.Status myStatus = io.opentelemetry.trace.Status.OK.withDescription("Status description");
+    Status spanStatus = TraceTranslator.toStatusProto(myStatus);
+
+    // The int representation is 0 for canonical code "OK".
+    assertEquals(0,myStatus.getCanonicalCode().value());
+    assertEquals(io.opentelemetry.trace.Status.CanonicalCode.OK, myStatus.getCanonicalCode());
+    assertEquals("Status description", spanStatus.getMessage());
+  }
+  
+  @Test
+  public void toLinksProtoTest(){
+    List<io.opentelemetry.sdk.trace.data.SpanData.Link> links = new ArrayList<>();
+
+    TraceId traceIdOne = new TraceId(321, 123);
+    SpanId spanIdOne = new SpanId(12345);
+    TraceFlags traceFlagsOne = TraceFlags.builder().build();
+    TraceState traceStateOne = TraceState.builder().build();
+    SpanContext spanContextOne = SpanContext.create(traceIdOne, spanIdOne, traceFlagsOne, traceStateOne);
+    Attributes attributesOne = Attributes.newBuilder().setAttribute("key", "value").build();
+    io.opentelemetry.sdk.trace.data.SpanData.Link linkOne =  io.opentelemetry.sdk.trace.data.SpanData.Link.create(spanContextOne, attributesOne);
+
+    TraceId traceIdTwo = new TraceId(32473, 24893);
```

Comment:
This block looks pretty similar to the above one, please consider to extract out a common method to reduce verbosity.

Reply:
Created a method to create Links to condense the code a bit/make it more readable